### DISCUSSION TITLE
Allow to choose spacing between taskbar icons

### DISCUSSION THREAD

**Discussion by glekner** [Open-Shell/Open-Shell-Menu#596] [Ideas]
Allow to choose spacing between taskbar icons, this would be a great addition that StartIsBack++ already has

**Comment by NebelNidas** [comment]
Not really configurable, but with [7+ Taskbar Tweaker](https://rammichael.com/7-taskbar-tweaker), you can remove the spacing: 
![image](https://user-images.githubusercontent.com/48808497/118880583-4f7b2900-b8f2-11eb-8ca3-697caed35193.png)

**Comment by GanerCodes** [comment]
I mean, startisback is able to do it, so I know its possible; remove extra gap doesn't even come close to the XS setting SIB++ has

**Reply by Gaurav-Original-ClassicShellTester** [reply]
For me at least, **Remove extra gap between items** has exactly the same spacing as StartIsBack Icon size Large, Icon Margins: XS. Pixel for pixel it was exactly the same when I measured it, after disabling StartIsBack and only testing the 7+TT setting.

What resolution and DPI are you running and what icon size for the taskbar icons?

**Reply by GanerCodes** [reply]
1920x1080, windows scaling 100% for my main monitor. Heres how it looks currently. When I used SIB+ they were nearly square
![image](https://user-images.githubusercontent.com/34655998/136810976-eeeb1038-7043-4489-b133-b81d020237c8.png)

**Reply by Gaurav-Original-ClassicShellTester** [reply]
You're right! I checked again on my other Windows 10 PC with only StartIsBack++ and only 7+TT! On this PC I have nothing pinned to the taskbar whereas on the other where I checked, I was comparing pinned items spacing.

For non-pinned items (with or without labels), SIB has much more compact spacing which is preferable for power users (top: SIB, bottom: 7+TT). My custom Open Shell animated Start button position is also shifted a little bit by it.

![image](https://user-images.githubusercontent.com/46422834/136846150-5dfac31f-6d5b-41c7-b00f-951c1902403b.png)

Sure would be nice to have in Open Shell.

**Reply by GanerCodes** [reply]
Any idea on how to fix it besides SIB?
